GitHubActions logger fails

I tried to use GitHubActions logger but it fails:

https://github.com/timholy/SnoopCompile.jl/runs/830320767
```julia
 [4] require(::Base.PkgId) at ./loading.jl:927
 [5] require(::Module, ::Symbol) at ./loading.jl:922
 [6] include(::String) at ./client.jl:439
 [7] top-level scope at /home/runner/work/SnoopCompile.jl/SnoopCompile.jl/test/runtests.jl:2
 [8] include(::String) at ./client.jl:439
 [9] top-level scope at none:6
in expression starting at /home/runner/work/SnoopCompile.jl/SnoopCompile.jl/test/snoopi.jl:1
in expression starting at /home/runner/work/SnoopCompile.jl/SnoopCompile.jl/test/runtests.jl:1
ERROR: Package SnoopCompile errored during testing
Stacktrace:
 [1] pkgerror(::String, ::Vararg{String,N} where N) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.4/Pkg/src/Types.jl:53
 [2] test(::Pkg.Types.Context, ::Array{Pkg.Types.PackageSpec,1}; coverage::Bool, julia_args::Cmd, test_args::Cmd, test_fn::Nothing) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.4/Pkg/src/Operations.jl:1510
 [3] test(::Pkg.Types.Context, ::Array{Pkg.Types.PackageSpec,1}; coverage::Bool, test_fn::Nothing, julia_args::Cmd, test_args::Cmd, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.4/Pkg/src/API.jl:316
 [4] #test#68 at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.4/Pkg/src/API.jl:297 [inlined]
 [5] #test#65 at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.4/Pkg/src/API.jl:294 [inlined]
 [6] top-level scope at none:1
##[error]The process 'julia' failed with exit code 1
```

--- a/README.md
+++ b/README.md
@@ -7,14 +7,25 @@ Utilities for working within GitHub Actions, modelled after [`actions/core`](htt
 
 Perhaps the most common use case is to set the global logger to one compatible with GitHub Actions' log format:
 
-```jl
-# Using GitHubActions logger in CI
+### Inside the package
+```
 using Logging: global_logger
 using GitHubActions: GitHubActionsLogger
 function __init__()
+    # Using GitHubActions logger in CI
     get(ENV, "GITHUB_ACTIONS", "false") == "true" && global_logger(GitHubActionsLogger())
 end
+```
+
+### Inside the runtests
+```jl
+using Logging: global_logger
+using GitHubActions: GitHubActionsLogger
+# Using GitHubActions logger in CI
+get(ENV, "GITHUB_ACTIONS", "false") == "true" && global_logger(GitHubActionsLogger())
 @warn "This warning will be visible in your GitHub Actions logs!"
 ```
+
+
 
 For information on the other provided functions, see the documentation.

--- a/README.md
+++ b/README.md
@@ -8,9 +8,12 @@ Utilities for working within GitHub Actions, modelled after [`actions/core`](htt
 Perhaps the most common use case is to set the global logger to one compatible with GitHub Actions' log format:
 
 ```jl
+# Using GitHubActions logger in CI
 using Logging: global_logger
 using GitHubActions: GitHubActionsLogger
-get(ENV, "GITHUB_ACTIONS", "false") == "true" && global_logger(GitHubActionsLogger())
+function __init__()
+    get(ENV, "GITHUB_ACTIONS", "false") == "true" && global_logger(GitHubActionsLogger())
+end
 @warn "This warning will be visible in your GitHub Actions logs!"
 ```
 

